package.json

```diff
@@ -77,7 +77,7 @@
     "typescript": "^4.1.2",
     "webpack": "^5.51.1",
     "webpack-cli": "^4.2.0",
-    "webpack-config-single-spa-react-ts": "^2.2.2",
+    "webpack-config-single-spa-react-ts": "^4.0.0",
     "webpack-dev-server": "^4.0.0-beta.0",
```

Comment:
update webpack-dev-server to the latest. also check the other single-spa dependencies are up-to-date